raiden/network/transport/matrix.py

```diff
@@ -127,14 +128,25 @@ class MatrixTransport:
     def __init__(self, config: dict):
         self._bound_logger = None
         self._raiden_service: RaidenService = None
+
+        def http_retry_delay() -> Union[float, Iterable[float]]:
+            if not self._raiden_service:
+                return 1
+            return udp_utils.timeout_exponential_backoff(
+                self._raiden_service.config['transport']['retries_before_backoff'],  # default = 5
+                self._raiden_service.config['transport']['retry_interval'] / 5,  # default=5/5=1s
+                self._raiden_service.config['transport']['retry_interval'],  # default=5s
```

Comment:
Instead of repeating the defaults here (and inevitably have them go out of sync with reality at some point) just add a comment pointing to where they are defined.